src/dataDisplay/icon/icon.jsx

```diff
@@ -1007,6 +1007,75 @@ class Icon extends React.PureComponent {
+                        '    M18.41,11.74c0.01,0.14,0.04,0.27,0.04,0.41l-0.01,0.11H24v-0.52H18.41z\n' +
+                        '    M20.65,9.41L18.1,10.1c0.05,0.16,0.11,0.33,0.15,0.5l2.53-0.68L20.65,9.41z\n' +
+                        '    M3.22,14.08l0.13,0.5l1.98-0.53c-0.05-0.17-0.12-0.32-0.15-0.5L3.22,14.08z\n' +
+                        '    M22.52,6.22l-0.26-0.45l-4.86,2.8c0.09,0.15,0.17,0.3,0.25,0.46L22.52,6.22z\n' +
+                        '    M1.48,17.78l0.26,0.45l4.33-2.5c-0.09-0.15-0.17-0.3-0.25-0.46L1.48,17.78z\n' +
+                        '    M16.35,7.29c0.13,0.12,0.24,0.25,0.36,0.38l1.86-1.86L18.2,5.44L16.35,7.29z\n' +
+                        '    M18.22,1.74l-0.45-0.26l-2.79,4.83c0.15,0.08,0.31,0.15,0.46,0.25L18.22,1.74z\n' +
+                        '    M14.59,3.35l-0.5-0.13l-0.66,2.46c0.17,0.04,0.33,0.10,0.50,0.15L14.59,3.35z\n' +
+                        '    M10.78,18.12c-1.09-1.65-0.28-3.02,1.09-4.58c0,2.6,1.65,2.8,1.35,4.58c3.19-2.39-0.17-4.56-0.17-9.57C7.36,13.09,8.4,17.48,10.78,18.12z\n' +
+                        '    M16.61,20.16H12.4c-0.01-0.32-0.07-0.9-0.31-1.49c-0.05-0.13-0.2-0.19-0.34-0.14c-0.13,0.05-0.2,0.20-0.14,0.34c0.20,0.51,0.26,1.01,0.27,1.3h-4.5c-0.5,0-0.86,0.47-0.74,0.95l0.6,2.32C7.34,23.77,7.64,24,7.99,24h8.02c0.35,0,0.65-0.23,0.74-0.57l0.6-2.32C17.47,20.63,17.11,20.16,16.61,20.16z',
+                    id: pathId,
+                };
+
+                viewBox = '0 0 23 23';
+
+                break;
+            case 'journey-crib':
+                path = {
+                    d: 'M10.68,6.27 9.58,7.92 11.23,6.82 12,11 12.77,6.82 14.53,7.92 13.32,6.27 17.5,5.5 13.32,4.73 14.53,3.08 12.77,4.18 12,0 11.23,4.18 9.58,3.08 10.68,4.73 6.5,5.5 \n' +
+                        'M18,12H6l5.2,7.7L8.3,24h1.6l2.1-3.1l2.1,3.1h1.6l-2.9-4.3L18,12z M8.3,13.1h7.3L12,18.5L8.3,13.1z\n' +
+                        'M10.1,13.97 12,16.87 13.9,13.97',
+                    id: pathId,
+                };
+
+                viewBox = '0 0 23 23';
+
+                break;
+            case 'journey-crown':
+                path = {
+                    d: 'M327.1,792c-0.7-0.1-1.4,0.1-1.5,0.6c-0.1,0.5,0.5,1,1.3,1.2c0.1,0,0.3,0,0.4,0.1\n' +
+                        '\t\t\tc-3.2,2-7.4,0.9-9.3-2.5c0,0,0,0,0,0c0.9-0.2,1.5-0.6,1.5-1c0-0.4-0.5-0.8-1.2-0.9c-0.2-1-0.7-2.3-1.3-2.3s-1.1,1.3-1.3,2.3\n' +
+                        '\t\t\tc-0.7,0.2-1.2,0.5-1.2,0.9c0,0.4,0.6,0.8,1.5,1c0,0,0,0,0,0c-1.9,3.4-6.1,4.5-9.3,2.5c0.1,0,0.3,0,0.4-0.1\n' +
+                        '\t\t\tc0.8-0.2,1.4-0.7,1.3-1.2c-0.1-0.4-0.7-0.7-1.5-0.6c-0.3-0.9-1.1-2.2-1.7-2.1c-0.4,0.1,0,2.8,0,2.8l1.4,7.4l20.8-0.1l1.4-7.3\n' +
+                        '\t\t\tc0,0,0.5-2.9,0-3C328.2,789.8,327.4,791.1,327.1,792z M327.2,801.3h-20.4c-0.4,0-0.8,0.3-0.8,0.7v0.2c0,0.4,0.3,0.7,0.8,0.7h20.4\n' +
+                        '\t\t\tc0.4,0,0.8-0.3,0.8-0.7V802C327.9,801.6,327.6,801.3,327.2,801.3',
```

Comment:
should we have the tabs `\t` chars there?

Reply:
No, that looks strange.  `\t` is an encoding of a tab, but we shouldn't need that.  Good catch!